pylatex/tikz.py

```diff
@@ -34,13 +46,466 @@ def __init__(self, options=None, *, data=None):
         super().__init__(options=options, data=data)
 
 
+class TikZScope(Environment):
+    """TikZ Scope Environment."""
+
+    _latex_name = 'scope'
+
+
+class TikZCoordinate(object):
```

Comment:
Hi, there was no specific reason for that, I just thought it unnecessary to subclass LatexObject. I will change it to that.